mars_troughs/accumulation_model.py

```diff
@@ -109,7 +110,8 @@ def get_accumulation_at_t(self, time: np.ndarray) -> np.ndarray:
             accumulation rates A, in m/year
 
         """
-        return self.intercept + self.slope * self._ins_data_spline(time)
+        return self.eval(time)
```

Comment:
@klaferr is it this line that's busted?

Reply:
No the line that causes the problem is the return line in get_yt, I get an error saying that there is no self.intercept if I remove the init call with that variable included.

Reply:
Ok shoot, I know why this is broken. I'll write up the solution now. The TLDR is that we are using "multiple inheritance" and the `__init__` function of the second parent class (`LagModel`) is not being called.

Reply:
I retract what I wrote before -- MI is not the problem. Can you clarify what tests fail? I pulled this branch and commented out lines 95 and 96, and all the tests still passed.